docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -237,7 +237,98 @@ Step 4: Confirm Network Functionality
         MyKey: 999
 
 
-Step 5: Stop the Sawtooth Network (Optional)
+.. _configure-txn-procs-docker-label:
+
+Step 5. Configure the Allowed Transaction Processors (Optional)
+---------------------------------------------------------------
+
+By default, a validator accepts transactions from any transaction processor.
+However, Sawtooth allows you to limit which transaction processors can submit
+transactions.
+
+In this step, you will configure the validator network to accept transactions
+only from the four transaction processors in the example environment:
+IntegerKey, Settings, XO, and Validator Registry. This on-chain setting is
+configured on one validator node, then Sawtooth sends the setting to the other
+nodes in the network as a transaction in a batch.
+
+The :doc:`Settings transaction processor
+<../transaction_family_specifications/settings_transaction_family>`
+handles on-chain configuration settings. You will use the ``sawset`` command to
+create and submit a batch of transactions containing the configuration change.
+
+Use the following steps to create and submit a batch containing the new on-chain
+setting.
+
+1. Connect to a validator container. The next command must use a key generated
```

Comment:
I think it is important what validator you connect to. By default, only the key used to sign the initial genesis transaction is authorized to change settings (whatever key was used when `sawset genesis` was run). So if this command was run from validator-0 using the default key, you would need to connect to that validator again or the transaction would be rejected.